thoth/metrics_exporter/jobs/security.py

```diff
@@ -54,8 +54,9 @@ def get_si_unanalyzed_python_packages_versions(cls) -> None:
 
         metric_name = "thoth_graphdb_total_number_si_unanalyzed_python_packages"
         metric = Configuration.PROM.get_current_metric_value(
-            metric_name=metric_name, label_config={"instance": cls._METRICS_EXPORTER_INSTANCE}
+            metric_name=metric_name, label_config={'field': cls._METRICS_EXPORTER_INSTANCE}
         )
+        print(metric)
```

Comment:
is this intentional?

Reply:
nice catch, I was testing the metric :) thanks!